### PR TITLE
Document how to trust specific CA certs for event senders

### PR DESCRIPTION
**Pull request by pierDipi**
As per title

### CODE REVIEW

File: docs/eventing/experimental-features/transport-encryption.md [hunk 1]
```diff
@@ -80,6 +80,99 @@ data:
   transport-encryption: "strict"
 ```
 
+## Trusting CA in a specific event sender
```
Comment (creydr) [inline docs/eventing/experimental-features/transport-encryption.md:83]:
```suggestion
## Trusting CA for a specific event sender
```
maybe?

Comment (pierDipi) [reply docs/eventing/experimental-features/transport-encryption.md]: Done

**Comment by creydr** [comment]
Only one comment/though. Otherwise LGTM from my side. Feel free to unhold

/lgtm
/hold

**Comment by pierDipi** [comment]
/unhold

We can iterate in case @matzew has comments

**Pull request merged**